From f06acc1ff0df91d265de03b19702292e24ed08a5 Mon Sep 17 00:00:00 2001
From: Yash Goyal <yash.goyal@pelion.com>
Date: Tue, 16 Mar 2021 00:04:27 -0700
Subject: [PATCH] Added trusted storage to Yocto target

parsec-se-driver will be added as DEPENDS in the recipe as its a rust
based project
---
 pal-platform/pal-platform.json | 30 +++++++++++++++---------------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/pal-platform/pal-platform.json b/pal-platform/pal-platform.json
index caf0e0e..9e50cb9 100644
--- a/pal-platform/pal-platform.json
+++ b/pal-platform/pal-platform.json
@@ -46,26 +46,26 @@
         "version": "1.0.0",
         "from": {
           "protocol": "git",
-          "location": "https://github.com/ARMmbed/psa_trusted_storage_linux.git",
+          "location": "https://github.com/PelionIoT/psa_trusted_storage_linux.git",
           "tag": "ptsl-0.0.1-psa-storage-api-1.0.0"
         },
         "to": "Middleware/trusted_storage/trusted_storage"
       },
       "parsec_se_driver": {
-        "version": "0.3.1",
+        "version": "0.4.0",
         "from": {
           "protocol": "git",
           "location": "https://github.com/parallaxsecond/parsec-se-driver.git",
-          "tag": "0.3.1"
+          "tag": "0.4.0"
         },
         "to": "Middleware/parsec_se_driver/parsec_se_driver"
       },
       "curl": {
-        "version": "7.72.0",
+        "version": "7.75.0",
         "from": {
           "protocol": "git",
           "location": "https://github.com/curl/curl.git",
-          "tag": "curl-7_72_0"
+          "tag": "curl-7_75_0"
         },
         "to": "Middleware/curl/curl"
       }
@@ -89,14 +89,14 @@
         },
         "to": "Middleware/mbedtls/mbedtls"
       },
-      "curl": {
-        "version": "7.72.0",
+      "trusted_storage": {
+        "version": "1.0.0",
         "from": {
           "protocol": "git",
-          "location": "https://github.com/curl/curl.git",
-          "tag": "curl-7_72_0"
+          "location": "https://github.com/PelionIoT/psa_trusted_storage_linux.git",
+          "tag": "ptsl-0.0.1-psa-storage-api-1.0.0"
         },
-        "to": "Middleware/curl/curl"
+        "to": "Middleware/trusted_storage/trusted_storage"
       }
     }
   },
@@ -118,11 +118,11 @@
         "to": "Middleware/mbedtls/mbedtls"
       },
       "curl": {
-        "version": "7.72.0",
+        "version": "7.75.0",
         "from": {
           "protocol": "git",
           "location": "https://github.com/curl/curl.git",
-          "tag": "curl-7_72_0"
+          "tag": "curl-7_75_0"
         },
         "to": "Middleware/curl/curl"
       }
@@ -213,17 +213,17 @@
         "version": "1.0.0",
         "from": {
           "protocol": "git",
-          "location": "https://github.com/ARMmbed/psa_trusted_storage_linux.git",
+          "location": "https://github.com/PelionIoT/psa_trusted_storage_linux.git",
           "tag": "ptsl-0.0.1-psa-storage-api-1.0.0"
         },
         "to": "Middleware/trusted_storage/trusted_storage"
       },
       "parsec_se_driver": {
-        "version": "0.3.1",
+        "version": "0.4.0",
         "from": {
           "protocol": "git",
           "location": "https://github.com/parallaxsecond/parsec-se-driver.git",
-          "tag": "0.3.1"
+          "tag": "0.4.0"
         },
         "to": "Middleware/parsec_se_driver/parsec_se_driver"
       }
-- 
2.17.1

